app/recordtransfer/templates/recordtransfer/submission_form_review.html

```diff
@@ -56,7 +58,9 @@
                                                         {% if forloop.counter > 5 %}
                                                             <div class="file-entry mb-2 flex items-center gap-2">
                                                                 <i class="fas fa-file text-primary/60"></i>
-                                                                <a class="non-nav-link text-primary/60 font-medium underline hover:text-primary transition"
+                                                                <a class="non-nav-link text-primary/60 font-medium underline hover:text-primary transition break-all block"
+                                                                   style="word-break: break-all;
+                                                                          max-width: 100%"
```

Comment:
Same comment as above

```suggestion
                                                                <a class="non-nav-link text-primary/60 font-medium underline hover:text-primary transition break-all max-w-full block"
```